rskj-core/src/main/java/co/rsk/validators/ValidTxExecutionSublistsEdgesRule.java

```diff
@@ -15,12 +17,22 @@
 public class ValidTxExecutionSublistsEdgesRule implements BlockValidationRule {
 
     private static final Logger logger = LoggerFactory.getLogger("blockvalidator");
+    private final ActivationConfig activationConfig;
+
+    public ValidTxExecutionSublistsEdgesRule(ActivationConfig activationConfig) {
+        this.activationConfig = activationConfig;
+    }
+
 
     @Override
     public boolean isValid(Block block) {
+        if (!activationConfig.isActive(ConsensusRule.RSKIP144, block.getHeader().getNumber())) {
+            return true;
```

Comment:
Even if execution is done by checking activation config I would add

```suggestion
            return edges == null;
```

Reply:
Not needed, it doesn't matter what comes in the txExecutionSublistsEdge because before RSKIP144, the code shouldn't take it into account.